src/bin/cross-util.rs

```diff
@@ -20,6 +20,20 @@ enum Commands {
     ListImages(commands::ListImages),
     /// Remove cross images in local storage.
     RemoveImages(commands::RemoveImages),
+    /// List cross data volumes in local storage.
+    ListVolumes(commands::ListVolumes),
+    /// Remove cross data volumes in local storage.
+    RemoveVolumes(commands::RemoveVolumes),
+    /// Prune volumes not used by any container.
+    PruneVolumes(commands::PruneVolumes),
+    /// Create a persistent data volume for the current crate.
+    CreateCrateVolume(commands::CreateCrateVolume),
+    /// Remove a persistent data volume for the current crate.
+    RemoveCrateVolume(commands::RemoveCrateVolume),
+    /// List cross containers in local storage.
+    ListContainers(commands::ListContainers),
+    /// Stop and remove cross containers in local storage.
+    RemoveContainers(commands::RemoveContainers),
```

Comment:
maybe these should be refactored into further subcommands?

`cross-util volumes prune` etc

Reply:
Sounds good, should probably be done with `images` too: `images list` and `images remove`.